source/common/upstream/eds.cc

```diff
@@ -118,10 +119,34 @@ void EdsClusterImpl::onConfigUpdate(const Protobuf::RepeatedPtrField<ProtobufWkt
                                      cluster_load_assignment.cluster_name()));
   }
 
+  // Reset assignment timeout (if set) as we have received new assignment.
```

Comment:
Can you add release notes to https://github.com/envoyproxy/envoy/blob/master/docs/root/intro/version_history.rst?